### PR TITLE
l10n: Triple dot to ellipsis

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

Signed-off-by: rakekniven <mark.ziegler@rakekniven.de>

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
I think @jancborchardt prefers to have "something …" over "something…" but not sure which one is correct

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
> I think @jancborchardt prefers to have "something …" over "something…" but not sure which one is correct

I don't know about English syntax, but French is no space and `...`
"Bonjour..." https://fr.wikipedia.org/wiki/Points_de_suspension :)

**Comment by skjnldsv** [comment]
Wow it's funny, in English, there is no defines rule, it varies a lot
https://en.wikipedia.org/wiki/Ellipsis
> The most common form of an ellipsis is a row of three periods or full points (...) or a precomposed triple-dot glyph (…). The usage of the em dash (—) can overlap the usage of the ellipsis, especially in dialogue. Style guides often have their own rules governing the use of ellipses. For example, The Chicago Manual of Style (Chicago style) recommends that an ellipsis be formed by typing three periods, each with a space on both sides ( . . . ), while the Associated Press Stylebook (AP style) puts the dots together, but retains a space before and after the group.[6]

**Comment by rakekniven** [comment]
Old story told again :-)

We need a definition and we need it to be written down.
Cannot find old discussion right now ...

**Comment by skjnldsv** [comment]
Let's use what was here before then, let's not go over it again for the sake of our poor Jan :see_no_evil: 

Before it was set to `…` (not on this code because it was old code, but in comments, uploads, tagging, theming...), space before, with a space before  :)

**Comment by ChristophWurst** [comment]
> Cannot find old discussion right now ...

Interesting use of ... over … :see_no_evil:

**Comment by rakekniven** [comment]
> Interesting use of ... over … 🙈

That was intentional. Very thoughtful.

**Comment by jancborchardt** [comment]
Just to confirm, "Something something …" is the form we use. That is, a space between last word and ellipsis.

Reason being that having no space looks like the last word is being ellipsized and cut off, which is not the case.

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Please change to " …" (space before ellipsis) for all, as mentioned. :) Thanks a lot!

**Comment by jancborchardt** [comment]
> Please change to " …" (space before ellipsis) for all, as mentioned. :) Thanks a lot!

@rakekniven do you want to make that change? :)

File: apps/files_sharing/src/components/SharingInput.vue [hunk 1]
```diff
@@ -122,16 +122,16 @@ export default {
 				return t('files_sharing', 'Resharing is not allowed')
 			}
 			if (!allowRemoteSharing && allowMailSharing) {
-				return t('files_sharing', 'Name or email address...')
+				return t('files_sharing', 'Name or email address…')
```
Comment (skjnldsv) [inline apps/files_sharing/src/components/SharingInput.vue]:
```suggestion
				return t('files_sharing', 'Name or email address …')
```

File: apps/files_sharing/src/components/SharingInput.vue [hunk 2]
```diff
@@ -122,16 +122,16 @@ export default {
...
 			}
 			if (allowRemoteSharing && !allowMailSharing) {
-				return t('files_sharing', 'Name or federated cloud ID...')
+				return t('files_sharing', 'Name or federated cloud ID…')
```
Comment (skjnldsv) [inline apps/files_sharing/src/components/SharingInput.vue]:
```suggestion
				return t('files_sharing', 'Name or federated cloud ID …')
```

File: apps/files_sharing/src/components/SharingInput.vue [hunk 3]
```diff
@@ -122,16 +122,16 @@ export default {
...
 			}
 			if (allowRemoteSharing && allowMailSharing) {
-				return t('files_sharing', 'Name, federated cloud ID or email address...')
+				return t('files_sharing', 'Name, federated cloud ID or email address…')
```
Comment (skjnldsv) [inline apps/files_sharing/src/components/SharingInput.vue]:
```suggestion
				return t('files_sharing', 'Name, federated cloud ID or email address …')
```

File: apps/files_sharing/src/components/SharingInput.vue [hunk 4]
```diff
@@ -122,16 +122,16 @@ export default {
...
 			}
 
-			return 	t('files_sharing', 'Name...')
+			return 	t('files_sharing', 'Name…')
```
Comment (skjnldsv) [inline apps/files_sharing/src/components/SharingInput.vue]:
```suggestion
			return 	t('files_sharing', 'Name …')
```

File: apps/files_sharing/src/components/SharingInput.vue [hunk 5]
```diff
@@ -147,7 +147,7 @@ export default {
 
 		noResultText() {
 			if (this.loading) {
-				return t('files_sharing', 'Searching...')
+				return t('files_sharing', 'Searching…')
```
Comment (skjnldsv) [inline apps/files_sharing/src/components/SharingInput.vue]:
```suggestion
				return t('files_sharing', 'Searching …')
```

**Review by jancborchardt — APPROVED** [review]
Or like that, thanks @skjnldsv @rakekniven! :)

**Pull request merged**